bin/search_fossasia.py

```diff
@@ -0,0 +1,27 @@
+#Returns all tweet text lines related to fossasia
+import urllib
+from urllib import request
+import html
+import json
+import sys
+
+query = "http://loklak.org/api/search.json?q=fossasia"
```

Comment:
Can you make the `q` parameter be a variable instead of hardcoding it to `fossasia` ? Take the `argv` values and you can do this.